internal/pkg/cli/interfaces.go

```diff
@@ -244,6 +253,14 @@ type environmentDeployer interface {
 	GetEnvironment(appName, envName string) (*config.Environment, error)
 }
 
+type taskResourceDeployer interface {
```

Comment:
nit: https://golang.org/doc/effective_go.html#interface-names
```suggestion
type taskDeployer interface {
```